debian/letsencrypt-doc.doc-base

```diff
@@ -0,0 +1,12 @@
+Document: letsencrypt-manual
```

Comment:
`letsencrypt-manual` might be easily confused with the manual authentication plugin (embedded in the core, rather than being distributed in a separate PyPI package). Could we have another name here?